vnext/Microsoft.ReactNative/Modules/DeviceInfoModule.h

```diff
@@ -3,6 +3,7 @@
 #pragma once
 
 #include <NativeModules.h>
+#include <React.h>
```

Comment:
It would be better if we use only the public APIs for the native modules. This way we can extract them into a separate DLL if needed. The React.h is the internal API which ideally we should reduce/remove in future.